tests/unit/test_persistent_routes.py

```diff
@@ -1,124 +0,0 @@
-import pytest
-from app.models import Resource, Language, Category
-from tests.conftest import module_client, module_db
-from configs import PaginatorConfig
-
-
-# TODO: We need negative unit tests (what happens when bad data is sent)
-def test_get_resources(module_client, module_db):
```

Comment:
Just to be clear, the reason I deleted `test_persistent_routes.py` is because with authentication required, we need to store the auth token in the database upon successful authentication, and that made every single "non-persistent" route persistent. Therefore, if every route needs a persistent database, then they're all the same and we may as well just call it `test_routes.py`. Technically, we don't need to have all the resources imported for every route, but we do need to have the auth token stored in the DB for create/update.

If there's a better way, let me know, but honestly, at this point I'm a fan of having all the routes be tested using the same test db